### PR TITLE
login user named on dashboard

### PR DESCRIPTION
**Pull request by shubhamdevgupta**

### CODE REVIEW

File: frontend/mgramseva/lib/screeens/landing_page/landing_page_new.dart [hunk 1]
```diff
@@ -29,82 +27,6 @@ class _LandingPageNewState extends State<LandingPageNew> {
...
-    'Karnataka',
-    'Kerala',
-    'Madhya Pradesh',
-    'Maharashtra',
-    'Manipur',
-    'Meghalaya',
-    'Mizoram',
-    'Nagaland',
-    'Odisha',
-    'Punjab',
```
Comment (anilsingha-eGov) [inline frontend/mgramseva/lib/screeens/landing_page/landing_page_new.dart:52]: Why state is removing ? @shubhamdevgupta

File: frontend/mgramseva/lib/screeens/landing_page/landing_page_new.dart [hunk 2]
```diff
@@ -29,82 +27,6 @@ class _LandingPageNewState extends State<LandingPageNew> {
...
-        break;
-      }
-    }
-    //items.shuffle();
-    setState(() {});
-  }
-
-  void shuffleList() {
-    for (int i = 0; i < state.length; i++) {
-      if (state.elementAt(i).contains(typedText)) {
```
Comment (anilsingha-eGov) [inline frontend/mgramseva/lib/screeens/landing_page/landing_page_new.dart:88]: why all this functions are removing ? @shubhamdevgupta

Comment (shubhamdevgupta) [reply frontend/mgramseva/lib/screeens/landing_page/landing_page_new.dart:88]: those are i added when i run in mobile application so now i will read state from asserts/state.json

File: frontend/mgramseva/lib/screeens/landing_page/landing_page_new.dart (hunk 1 above)
Comment (shubhamdevgupta) [reply frontend/mgramseva/lib/screeens/landing_page/landing_page_new.dart:52]: this is static image we created for mobile now we will take this from asserts/json file

Comment (anilsingha-eGov) [reply frontend/mgramseva/lib/screeens/landing_page/landing_page_new.dart:52]: Okay @shubhamdevgupta

**Review by anilsingha-eGov — APPROVED** [review]

**Pull request merged**